python/housinginsights/sources/base.py

```diff
@@ -145,7 +145,7 @@ def result_to_csv(self, fields, results, filepath):
             writer = csv.writer(f, delimiter=',')
             writer.writerow(fields)
             for result in results:
-                writer.writerow(result.data)
+                writer.writerow(result)
```

Comment:
@ajhalani This actually originated in the `mar.py` class, which was the first one written - hence why @jkwening used the same method in `DCHousing`. I agree that result_to_csv data structure was a bit confusing. I'll make the same change to mar.py after merging